ql/src/java/org/apache/hadoop/hive/ql/optimizer/stats/annotation/StatsRulesProcFactory.java

```diff
@@ -2921,6 +2920,77 @@ public Object process(Node nd, Stack<Node> stack, NodeProcessorCtx procCtx,
     }
   }
 
+  /**
+   * LateralViewJoinOperator joins the output of select with the output of UDTF.
```

Comment:
Could you provide a bit more details about what the rule does? Most of the other rules in this class give a general overview of the cost model they implement.

Reply:
@zabetak Thanks for taking a look!
I added a description. Please feel free to ask me if something doesn't make sense.